screens/product.html

```diff
@@ -0,0 +1,1189 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Add pointer on cursor hover

![image](https://user-images.githubusercontent.com/91944618/154856185-62071613-24f9-49ce-8329-a9724ef5903b.png)

Reply:
Thanks @PuneetP16  Just added the cursor pointer .